content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md

```diff
@@ -0,0 +1,163 @@
+
+This month, Dynatrace’s [Henrik Rexed](https://github.com/henrikrexed) spoke with J, who works at a financial services organization, about how they use OpenTelemetry with GraphQL.
+
+## Overview
+
+J and his team embarked on their OpenTelemetry journey for two main reasons:
+
+* J’s company uses a few different observability back-ends. His team had switched to a vendor back-end that was different from the back-end used by other teams that they interfaced with. OpenTelemetry allowed them to continue to get end-to-end Traces in spite of using different vendors.
+* His team was using GraphQL, and needed to be able to better understand what was happening behind the scenes with their GraphQL calls.
+
+J also shared:
+
+* His team’s OpenTelemetry setup
+* How he and his team have helped other teams start using OpenTelemetry
+* His quest to make OpenTelemetry a standard at his organization
+* Challenges that he and his team encountered in their OpenTelemetry journey, along with a few suggestions for improvement.
+
+## Q&A
+
+### Why OpenTelemetry?
+
+J’s company has a diverse tech ecosystem, ranging from on-premise old-school mainframes, to AWS Cloud and Azure Cloud, where they run both Windows and Linux servers. They also use a number of different languages, including [Node.JS](https://nodejs.org/en/), [.NET](https://dotnet.microsoft.com/en-us/), [Java](https://www.java.com/en/), C, C++, and [PL/I](https://en.wikipedia.org/wiki/PL/I) (mainframe).
+
+Across the organization, different teams have chosen to use different observability platforms to suit their needs, resulting in a mix of both open source and proprietary observability tools.
+
+J’s team had recently migrated from one observability back-end to another. After this migration, they started seeing gaps in trace data, because other teams that they integrated with were still using a different observability back-end. As a result, they no longer had an end-to-end picture of their traces. The solution was to use a standard, vendor-neutral way to emit telemetry: OpenTelemetry.
+
+Another reason that his team took to using OpenTelemetry was [GraphQL](https://graphql.org), which they had been using for four years. [GraphQL](https://en.wikipedia.org/wiki/GraphQL) is an open-source language used to query and manipulate APIs. With GraphQL, everything is held in the body of data: request, response and errors, and as a result everything returns an [HTTP status of 200](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200), giving the impression that even failures are successful. This meant that J and his team had no visibility into what was going on behind the scenes.
```

Comment:
I find it odd to link to 2 different places with the same word, I suggest to drop the link to the wikipedia page.

Reply:
Right. The oddness comes from the fact that the same anchor text is used. I'd vote to keep the link but change the anchor text:

> GraphQL is an [open-source language used to query and manipulate APIs](https://en.wikipedia.org/wiki/GraphQL).

WDYT?

Reply:
I'm good with your suggestion @chalin!